hwcomposer/hwcomposer_backend_v11.cpp

```diff
@@ -357,34 +651,34 @@ HwComposerBackend_v11::sleepDisplay(bool sleep)
         m_vsyncTimeout.stop();
         hwc_device->eventControl(hwc_device, 0, HWC_EVENT_VSYNC, 0);
 
-#ifdef HWC_DEVICE_API_VERSION_1_4
-        if (hwc_version == HWC_DEVICE_API_VERSION_1_4) {
-            HWC_PLUGIN_EXPECT_ZERO(hwc_device->setPowerMode(hwc_device, 0, HWC_POWER_MODE_OFF));
-        } else
-#endif
-#ifdef HWC_DEVICE_API_VERSION_1_5
-        if (hwc_version == HWC_DEVICE_API_VERSION_1_5) {
-            HWC_PLUGIN_EXPECT_ZERO(hwc_device->setPowerMode(hwc_device, 0, HWC_POWER_MODE_OFF));
-        } else
-#endif
-            HWC_PLUGIN_EXPECT_ZERO(hwc_device->blank(hwc_device, 0, 1));
+        for (int i=0; i<num_displays; i++) {
+            if (g_unblanked_displays[i]) {
+                blankDisplay(i, true);
```

Comment:
As terms go personally I like blanking more than sleeping, but ideally whould stick to one term and at least use it consistently within the scope of internal variables. Now display on/off -> sleepDisplay true/false -> blankDisplay true/false -> device setPowerMode on/off.